definitions/consented/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -3108,11 +3108,155 @@
     "CaseFieldID": "civilPartnership",
     "PageFieldDisplayOrder": 2,
     "DisplayContext": "OPTIONAL",
-    "PageID": "amendCase",
+    "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1,
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "FinancialRemedyMVP2",
+    "CaseEventID": "FR_amendCase",
+    "CaseFieldID": "natureOfApplication1",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 6,
```

Comment:
I am wondering why `PageDisplayOrder` isn't 2

Reply:
We're going from a LiveFrom of when you committed for now.

Reply:
Updated numbering